Localization/pl-PL.hjson

```diff
@@ -218,22 +218,22 @@ Mods: {
 		}
 
 		BuffName: {
-			//CriticalJudgement: "Critical Judgement"
-			//HackAndSlash: "Hack and Slash"
+			CriticalJudgement: "Krytyczne Myślenie"
```

Comment:
This doesn't sound right, you're translating a vaguely unintentional pun. The main meaning is that the player is judging the enemy and also inflicting revenge upon them, so the second word should be a translation of "sentence" instead

Reply:
I see, I assumed judgement here meant "assessment". Then i could change it to "Osąd", meaning judgement in the context of court. Would that work? And in this case, i'll also change the descrition to mean "unleash your sentence".